tests/integration/hosts/aftereffects/test_deadline_publish_in_aftereffects.py

```diff
@@ -0,0 +1,79 @@
+import logging
+
+from tests.lib.assert_classes import DBAssert
+from tests.integration.hosts.aftereffects.lib import AEDeadlinePublishTestClass
+
+log = logging.getLogger("test_publish_in_aftereffects")
+
+
+class TestDeadlinePublishInAfterEffects(AEDeadlinePublishTestClass):
+    """Basic test case for DL publishing in AfterEffects
+
+        Uses generic TestCase to prepare fixtures for test data, testing DBs,
+        env vars.
+
+        Opens AfterEffects, run DL publish on prepared workile.
+
+        Test zip file sets 3 required env vars:
+        - HEADLESS_PUBLISH - this triggers publish immediately app is open
+        - IS_TEST - this differentiate between regular webpublish
+        - PYBLISH_TARGETS
+
+        Waits for publish job on DL is finished.
+
+        Then checks content of DB (if subset, version, representations were
+        created.
+        Checks tmp folder if all expected files were published.
+
+    """
+    PERSIST = False
+
+    TEST_FILES = [
+        ("1xhd2ij2ixyjCyTjZgcJEPAIiBHLU1FEY",
+         "test_aftereffects_publish.zip",
+         "")
+    ]
+
+    APP = "aftereffects"
```

Comment:
I would preffer `HOST_NAME` instead of `APP`.

Reply:
Ok, renamed.